static/templates/settings/organization-settings-admin.handlebars

```diff
@@ -49,6 +51,28 @@
         {{t "Only admins may invite" }}
       </label>
     </div>
+    <div class="input-group {{#unless server_inline_image_preview}}control-label-disabled{{/unless}}">
+      <input type="checkbox" class="inline-block" id="id_realm_inline_image_preview" name="realm_inline_image_preview"
+        {{#unless server_inline_image_preview}}disabled="disabled"{{/unless}}
+        {{#if realm_inline_image_preview}}checked="checked"{{/if}} />
+      <label for="id_realm_inline_image_preview"
+        id="id_realm_inline_image_preview_label"
+        title="{{t 'If checked, image preview will be shown.' }}"
+        class="inline-block">
+        {{t "Show previews of uploaded and linked images" }}
+      </label>
+    </div>
+    <div class="input-group {{#unless server_inline_url_embed_preview}}control-label-disabled{{/unless}}">
+      <input type="checkbox" class="inline-block" id="id_realm_inline_url_embed_preview" name="realm_inline_url_embed_preview"
+        {{#unless server_inline_url_embed_preview}}disabled="disabled"{{/unless}}
+        {{#if realm_inline_url_embed_preview}}checked="checked"{{/if}} />
+      <label for="id_realm_inline_url_embed_preview"
+        id="id_realm_inline_url_embed_preview_label"
+        title="{{t 'If checked, url embed preview will be shown.' }}"
+        class="inline-block">
+        {{t "Show previews for links" }}
+      </label>
+    </div>
```

Comment:
I think you just want to put the entire `<div>` block into the `{{#unless server_inline_url_embed_preview}}` block, not disable the input in that case. This applies to both.